.github/workflows/build.yaml

```diff
@@ -1,26 +1,363 @@
+          run: cp -a build/debian build/wake-*
+
+        - name: Build docker image
+          run: docker build -f dockerfiles/${{ matrix.dockerfile }} ${{ matrix.extra_docker_build_args }} -t wake-${{ matrix.dockerfile }} .
+
+        - name: Build
+          run: docker run --rm --mount type=bind,source=$PWD/build,target=/build --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined ${{ matrix.extra_docker_run_args }} wake-${{ matrix.dockerfile }} /bin/sh -c '${{ matrix.build_cmd }}'
+
+        - name: Test
+          run: docker run --rm --mount type=bind,source=$PWD/build,target=/build --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined wake-${{ matrix.dockerfile }} /bin/sh -c '${{ matrix.test_cmd }}'
+
+        - name: Install
+          run: install -D -t release/${{ matrix.target }} ${{ matrix.install_src_glob }}
+
+        - name: Upload
+          uses: actions/upload-artifact@v3
+          with:
+            name: release_${{ matrix.target }}
+            path: release/${{ matrix.target }}
+
+    macos_10_15:
+        name: MacOS 10.15 Build
         runs-on: macos-10.15
 
         steps:
-            - name: "Clone wake"
+            - name: Clone wake
               uses: actions/checkout@v2
               with:
                 fetch-depth: 0
 
-            - name: "Install deps"
+            - name: Install deps
               run: |
                 brew install --cask osxfuse
                 brew install re2 || brew install re2 --head
 
-            - name: "Build wake"
+            - name: Build wake
               run: make all -j3
 
-            - name: "Test wake"
+            - name: Test wake
               run: make test
+
+    docs:
+      runs-on: ubuntu-latest
+      needs: tarball
+      name: Docs
+      steps:
+        - name: Download index.html
+          uses: actions/download-artifact@v3
+          with:
+            name: index.html
+
+        - name: Download html
+          uses: actions/download-artifact@v3
+          with:
+            name: html
+
+        - name: Make .nojekyll
+          run: touch .nojekyll
+
+        - name: Upload
+          uses: actions/upload-pages-artifact@v1
+          with:
+            path: .
+
+    deploy_docs:
+      # Only run the 'deploy_docs' job if this workflow was triggered by a push to master
+      if: github.event_name == 'push' && github.ref_type == 'branch' && github.ref_name == 'master'
+
+      permissions:
+        pages: write
+        id-token: write
+
+      environment:
+        name: github-pages
+        url: ${{ steps.deployment.outputs.page_url }}
+
+      runs-on: ubuntu-latest
+      needs: docs
+      name: Deploy Docs to GH Pages
+      steps:
+        - name: Deploy
+          id: deployment
+          uses: actions/deploy-pages@v1
+
+    release:
+      # Only run the 'release' job if this workflow was triggered by the creation of a tag
+      if: github.event_name == 'push' && github.ref_type == 'tag'
```

Comment:
I foresee a need to update the "Make a release" workflow docs.  That's probably a good thing; this *feels* like it should allow a bit easier releases.

Reply:
Yep, I'm planning on making the update after this lands. I need to land this before update the docs to make sure I get the details right since I can only test the full flow after it lands,